test/testhelper/common.go

```diff
@@ -0,0 +1,141 @@
+package testhelper
```

Comment:
this is like a refactor of the existing test util funcs. We should put this in the separated PR and refactor the existing code if needed.

Reply:
Agree, the refactor on legacy e2e is on the way. I will have several little PRs to do that. But I don't think this will be a blocker for this PR, we can do 2 in parallel.